test/opsrc/catalog_source.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1
```

Comment:
Name of the file should be `catalogsourceconfig.yaml`

Reply:
Thanks @sbose78, good catch! I renamed the file.